apps/docs/content/tokens/semantic/color.mdx

```diff
@@ -8,10 +8,314 @@ import tokensDark from '../../../datas/tokens-dark.json';
 export const categoryKey = "color";
 
 export const tabs = [
-  { title: "Light", data: tokens.semantic[categoryKey], category: categoryKey },
-  { title: "Dark", data: tokensDark.semantic[categoryKey], category: categoryKey },
+  { title: "Light", category: categoryKey },
+  { title: "Dark", category: categoryKey }
 ];
 
+
 # Semantic Color
+<Tabs tabs={tabs}>
+  <div>
+    ## Neutral
+    <TableSection categoryKey={categoryKey} tokens={tokens.semantic[categoryKey]} categories={
+      ["hop-neutral"]
+    } />
+    ## Primary
+    <TableSection categoryKey={categoryKey} tokens={tokens.semantic[categoryKey]} categories={
+      ["hop-primary"]
+    } />
+    ## Success
+    <TableSection categoryKey={categoryKey} tokens={tokens.semantic[categoryKey]} categories={
+      ["hop-success"]
+    } />
+    ## Warning
+    <TableSection categoryKey={categoryKey} tokens={tokens.semantic[categoryKey]} categories={
+      ["hop-warning"]
+    } />
+    ## Danger
+    <TableSection categoryKey={categoryKey} tokens={tokens.semantic[categoryKey]} categories={
+      ["hop-danger"]
+    } />
+    ## Information
+    <TableSection categoryKey={categoryKey} tokens={tokens.semantic[categoryKey]} categories={
+      ["hop-information"]
+    } />
+    ## Upsell
+    <TableSection categoryKey={categoryKey} tokens={tokens.semantic[categoryKey]} categories={
+      ["hop-upsell"]
+    } />
+
+    ## Decorative
+    ### Decorative - Option 1
```

Comment:
the Decorative here is repetitive because we're in the **Decorative** section I'd remove it everywhere

Reply:
ok I kept it just because when you're scrolling I thought it may not be obvious where you are. But I can remove it.

Reply:
we could add a table of content on the side to help search the page

<img width="967" alt="Capture d’écran, le 2023-10-31 à 16 04 11" src="https://github.com/gsoft-inc/wl-hopper/assets/2379339/5c916842-2e1d-445d-898a-5e4f14706afd">